lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1887,8 +1915,9 @@ OpFoldResult Aten__Contains__IntListOp::fold(ArrayRef<Attribute> operands) {
 
 using BinaryIntOperatorFn = std::function<int64_t(int64_t, int64_t)>;
 template <typename OpTy>
-static OpFoldResult atenBinaryIntOperatorFoldHelper(OpTy op,
-                                                    BinaryIntOperatorFn f) {
+static std::enable_if_t<!std::is_same_v<OpTy, ArrayRef<Attribute>>,
+                        OpFoldResult>
+atenBinaryIntOperatorFoldHelper(OpTy op, BinaryIntOperatorFn f) {
```

Comment:
You should be able to unify these with just `ArrayRef<Attribute> operands` (this will require changing some of the other folders)

Reply:
Done.